e2e/atlas/integrations_test.go

```diff
@@ -100,15 +109,7 @@ func TestIntegrations(t *testing.T) {
 
 		var thirdPartyIntegrations mongodbatlas.ThirdPartyIntegrations
 		if err := json.Unmarshal(resp, &thirdPartyIntegrations); a.NoError(err) {
-			found := false
-			services := thirdPartyIntegrations.Results
-			for i := range services {
-				if services[i].Type == flowdockEntity {
-					found = true
-					break
-				}
-			}
-			a.True(found)
+			integrationExists(flowdockEntity, thirdPartyIntegrations)
```

Comment:
the error is you never use the return value of the function, here you're missing a `a.True(integrationExists(...))`